packages/calypso-e2e/src/lib/blocks/image-block.ts

```diff
@@ -32,13 +30,14 @@ export class ImageBlock {
 	/**
 	 * Uplaods the target file at the supplied path to WPCOM.
 	 *
-	 * @param {string} path Path to the file on disk.
+	 * @param {string} filePath Path to the file on disk.
 	 */
-	async upload( path: string ): Promise< void > {
+	async upload( filePath: string ): Promise< void > {
 		const input = await this.block.waitForSelector( selectors.fileInput, { state: 'attached' } );
-		await input.setInputFiles( path );
+		await input.setInputFiles( filePath );
+
 		await Promise.all( [
-			this.block.waitForSelector( selectors.spinner, { state: 'hidden' } ),
+			this.block.waitForSelector( 'img:not([src^="blob:"])' ),
```

Comment:
Side note: interesting selector - can you tell me more about this?

Reply:
Will add that comment above the selector!

Reply:
https://github.com/Automattic/wp-calypso/pull/58546/commits/bd49c0d79f01173846dfba3d68c18ea19d50ad49